pinot-core/src/test/java/org/apache/pinot/queries/InnerSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -49,27 +49,49 @@
       " GROUP BY column1, column3, column6, column7, column9, column11, column12, column17, column18";
 
   @Test
-  public void testAggregationOnly() {
```

Comment:
Don't remove the existing `testAggregationOnly()`. Since we added a new `FilteredAggregationsTest`, not sure if we still want to add this test. If so, let's keep them as 2 separate tests